tech-team-practices.md

```diff
@@ -0,0 +1,40 @@
+# Staff sync practices
```

Comment:
```suggestion
# Tech team internal coordination process
```

We should communicate that:

1. This is for the tech team
2. Once we have volunteers, will they also participate?
3. The goal of this is to co-ordinate between the teams, not to 'report up'.

I'm not sold on my suggestion either, would love to iterate.